build/sections/founder.html

```diff
@@ -0,0 +1,24 @@
+<div class="container">
+    <div class="pt-36">
+        <div class="flex flex-col w-full justify-center md:justify-start text-white font-raleway md:w-[80%] py-5">
+            <h2 class="font-bold text-2xl md:text-4xl font-raleway">Die <span class="text-primary-green">Founder</span> und ihre Vision von Ojemba</h2>
+            <div class="py-10 text-base font-montserrat font-medium w-full md:w-[90%] lg:w-[75%] 2xl:w-[60%]">
```

Comment:
Avoid using arbitrary values, but use the equivalent builtin utility tailwind classes. 
Example: Instead of `w-75%` you could use `w-3/4`, same on `w-60%`.

Reply:
Completed 💯